### DISCUSSION TITLE
I want better styling for inputs and selects

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#167] [Wants]
We all have been there: Designers want big, colorful checkboxes, radio buttons and selects. For the sake of accessibility it would be awesome if these elements could be styled with CSS in a better way. How often do we see something like `*-appearance: none;` and a circled `:before`. Or—slightly better—playing around with the height, the width and the opacity of the input. More accessible, but still a workaround. With the first solution these elements are not part of the accessibility tree—so make them fully tailorable.

<hr>

https://webwewant.fyi/wants/106/

**Comment by carmacleod** [comment]
In case it's helpful, see: https://open-ui.org/
> The purpose of Open UI to the web platform is to allow web developers to style and extend built-in web UI controls, such as <select> dropdowns, checkboxes, radio buttons, and date/color pickers.

**Reply by aarongustafson** [reply]
Yes! Thank you! We are tracking that as it evolves. I’ll get it added to the related links for this one.

**Comment by Crissov** [comment]
Meanwhile, [CSS Form Control Styling Module Level 1](https://drafts.csswg.org/css-forms/) has been drafted.
